client/dashboard/me/monetize-subscriptions/monetize-item.tsx

```diff
@@ -0,0 +1,87 @@
+import { MonetizeSubscription } from '@automattic/api-core';
+import { siteByIdQuery } from '@automattic/api-queries';
+import { formatCurrency } from '@automattic/number-formatters';
+import { useQuery } from '@tanstack/react-query';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import { formatDate } from 'date-fns';
+import SiteIcon from '../../sites/site-icon';
+
+export const MonetizeSubscriptionTerms = ( {
+	subscription,
+}: {
+	subscription: MonetizeSubscription;
+} ) => {
+	if ( subscription.end_date === null ) {
+		return <>{ __( 'Never expires' ) }</>;
+	}
+
+	return (
+		<>
+			{ subscription.renew_interval === null
+				? // eslint-disable-next-line @wordpress/i18n-translator-comments
+				  sprintf( __( 'Expires on %(date)s' ), {
+						date: formatDate( subscription.end_date, 'LL' ),
+				  } )
+				: // eslint-disable-next-line @wordpress/i18n-translator-comments
+				  sprintf( __( 'Renews at %(amount)s on %(date)s' ), {
+						amount: formatCurrency( Number( subscription.renewal_price ), subscription.currency ),
+						date: formatDate( subscription.end_date, 'LL' ),
+				  } ) }
+		</>
+	);
+};
+
+export const MonetizeSiteLink = ( { subscription }: { subscription: MonetizeSubscription } ) => {
+	const siteUrl = subscription.site_url.replace( /^https?:\/\//, '' );
+
+	return (
+		<button
+			onClick={ ( event ) => {
+				event.stopPropagation();
+				event.preventDefault();
+				window.location.href = subscription.site_url;
+			} }
+			title={ String(
+				// eslint-disable-next-line @wordpress/i18n-translator-comments
+				sprintf( __( 'Visit %(siteUrl)s' ), {
+					siteUrl: subscription.site_url,
+				} )
```

Comment:
I wonder if we could add the translator comments instead of disabling the linter, something like:

```suggestion
				sprintf(
					// translators: %(siteUrl)s is the URL of the site
					__( 'Visit %(siteUrl)s' ),
					{
						siteUrl: subscription.site_url,
					}
				)
```

Reply:
Done in fade306d07efdcc81a9a1ede26d451112ab2dcb4